octree.h

```diff
@@ -2301,7 +2312,8 @@ namespace OrthoTree
 
 
     // Update id with rebalancing by the new point information
-    bool Update(std::size_t entityID, TVector const& newPoint, std::span<TVector const> const& points) noexcept
+    template<AccessorConcept TAccessor = std::span<const TVector>>
+    bool Update(std::size_t entityID, TVector const& newPoint, TAccessor const& points) noexcept
```

Comment:
`std::size_t` could be replaced with `TAccessor::key_type` (to be easily replaceable with `std::unordered_map`)